coap/idf_component.yml

```diff
@@ -1,4 +1,4 @@
-version: "4.3.4~3"
+version: "4.3.4~4"
```

Comment:
Thanks @mrdeep1 for the PR

I think we should change it to `4.3.5-rc1`. I think component manager will accept that version. Using 4.3.5-rc1 will make it consistent with upstream.

Reply:
What goes in `coap/sbom_libcoap.yml` for the version?
````
name: libcoap
version: 4.3.4
cpe: cpe:2.3:a:libcoap:libcoap:{}:*:*:*:*:*:*:*
supplier: 'Organization: libcoap <https://libcoap.net/>'
description: A CoAP (RFC 7252) implementation in C
url: https://github.com/obgm/libcoap
hash: 0392d022173516b6d33a95b8285ba3cbc8767bdf
````

Reply:
Updated `coap/idf_component.yml `only to see what happens.

Reply:
I think we will need to update the **version** in sbom_libcoap.yml file as well.

Reply:
BTW, what is the timeline for **v4.3.5** release?

Reply:
All depends on what shaking of the v4.3.5rc1 tree finds.  Should be mid July latest I guess.